src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,28 @@
-export const Tabs = () => {};
+import { Tab } from '../Tab/Tab';
+
+export const Tabs = ({ tabs, onTabSelected, activeTabId }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId.id);
```

Comment:
The `activeTab` should be determined by comparing `tab.id` directly to `activeTabId`, not `activeTabId.id`. Change the comparison to `tab.id === activeTabId`.